docs/guide/README.md

```diff
@@ -13,7 +13,7 @@ MetaMask was created out of the needs of creating more secure and usable Ethereu
 
 - [Get started here](/guide/getting-started.html)
 - [Read the full docs of our injected provider](/guide/ethereum-provider.html)
-- [Read the full docs of the JSON RPC API](/guide/json-rpc-api.html)
+- [Read the full docs of the JSON RPC API](/guide/rpc-api.html)
```

Comment:
I'm guessing this title should be updated as well:
```suggestion
- [Read the full docs of the RPC API](/guide/rpc-api.html)
```

Reply:
Good catch, I found a few more similar references that I updated.